### DISCUSSION TITLE
Removing wrapper around component's children

### DISCUSSION THREAD

**Discussion by matthewmichel** [GrapesJS/grapesjs#4667] [Q&A]
I have a container component that I've created and added to my Grapes editor. Each of these children have styles that require them to be direct children of the parent Container component, but Grapes adds a <span> wrapper that contains all the children components. How do you disable this <span> wrapper? Is that possible with a custom View type?

Thanks

**Comment by artf** [comment]
Can you show an example of your custom component?
